calculator/operations.py

```diff
@@ -1,23 +1,56 @@
+from calculator.exceptions import *
 
```

Comment:
This applies to all your operations, you're already doing validity checking on parameters in your `perform_operation` in the calculator file. You don't need to do it twice as the parameters should already have been validated by the time it gets to this code.